### ISSUE TITLE
Setting personal alias doesn't work

### ISSUE THREAD

**Issue by phochste** [LibreCat/LibreCat#40]
Login as normal user. Click on the text 'No Author IDs set' next to your avatar. Fill in a text in Alias. Press save. 

Gives an 404 with the error

```(json)
{
  "error": "Serializer (Dancer::Serializer::JSON) failed at serializing http://localhost:5001/librecat:\nencountered object 'http://localhost:5001/librecat', but neither allow_blessed, convert_blessed nor allow_tags settings are enabled (or TO_JSON/FREEZE method missing) at (eval 563) line 151.\n"
}
```

**Comment by vpeil** [comment]
I get the same error, but the value is stored.

**Comment by vpeil** [comment]
fixed in eee804a31b48551eff8e515c5e5a0ea0c349818d
